partiql-ast/src/ast.rs

```diff
@@ -401,6 +402,8 @@ pub enum Expr {
     Path(AstNode<Path>),
     Call(AstNode<Call>),
     CallAgg(AstNode<CallAgg>),
+    /// <expr> MATCH <graph_pattern>
+    GraphMatch(AstNode<GraphMatch>),
```

Comment:
Nice, modeling looks very similar to what's currently in PLK

Reply:
Well, I did add develop both of them together 😄  :
- https://github.com/partiql/partiql-lang-rust/pull/148
- https://github.com/partiql/partiql-lang-kotlin/pull/652
- https://github.com/partiql/partiql-lang-kotlin/pull/658